src/components/AddEvent/AddEvent.js

```diff
@@ -1,117 +1,113 @@
-import { StyledButton, Title } from "../../assets/styles/shared.styled";
-import { CreateEvent, InputWrapper } from "./AddEvent.styled";
-import { useFormik } from "formik";
+import { useState } from "react";
 
-const AddEvent = () => {
-  const formik = useFormik({
-    initialValues: {
-      title: "",
-      attendees: "",
-      location: "",
-      description: "",
-      date: "",
-      time: "",
-    },
-    onSubmit: (values) => {
-      console.log("Form data", values);
-      formik.resetForm();
-    },
-    validate: (values) => {
-      let errors = {};
+import axios from "axios";
+import { Formik, Form } from "formik";
+import * as Yup from "yup";
 
-      if (!values.title) {
-        errors.title = "Title is required";
-      }
+import { Button, Grid, Stack } from "@mui/material";
 
-      if (!values.date) {
-        errors.date = "Date is required";
-      }
+import { CreateEvent } from "./AddEvent.styled";
+import { Title } from "../../assets/styles/shared.styled";
+
+import TextFieldWrapper from "./FormsUI/TextField";
+import DateTimePicker from "./FormsUI/DateTimePicker";
+import ButtonWrapper from "./FormsUI/Button";
+
+const initialValues = {
+  title: "",
+  attendees: "",
+  location: "",
+  description: "",
+  date: "",
+  time: "",
+};
 
-      if (!values.time) {
-        errors.time = "Time is required";
+const validationSchema = Yup.object({
+  // complex validation to be added
+  title: Yup.string().required("Title is required"),
+  attendees: Yup.string().required("Attendees are required"),
+  location: Yup.string().required("Location is required"),
+  date: Yup.string().required("Date is required"),
+  time: Yup.string().required("Time is required"),
+});
+
+const onSubmit = (values, { resetForm }) => {
+  axios
+    .post(
+      "https://react-events-app-7e674-default-rtdb.europe-west1.firebasedatabase.app/events.json",
+      {
+        title: `${values.title}`,
+        attendees: `${values.attendees}`,
+        location: `${values.location}`,
+        description: `${values.description}` || "No description provided",
+        date: `${values.date}`,
+        time: `${values.time}`,
       }
+    )
+    .then((response) => {
+      console.log(response);
```

Comment:
sometimes is ok to return response, to see when the request is finished so you can display a confirmation on the screen (or to redirect the user to other page). So remove console.log